uniswap-v2/logics/impls/router/data.rs

```diff
@@ -6,4 +7,6 @@ pub const STORAGE_KEY: u32 = openbrush::storage_unique_key!(Data);
 #[openbrush::upgradeable_storage(STORAGE_KEY)]
 pub struct Data {
     pub factory: AccountId,
+    // TODO: remove pair code hash as like https://github.com/Uniswap/v2-periphery/blob/master/contracts/libraries/UniswapV2Library.sol#L18
+    pub pair_code_hash: Hash,
```

Comment:
I think you will need to keep this in storage.
But add a setter with onlyOwner to change it in case of a contract upgrade.

Reply:
Got it. For now, let's not add a setter since our code base doesn't support upgrade currently.

Reply:
I think you can remove the comment line